ops/terraform/services/migrator/deploy.groovy

```diff
@@ -24,11 +23,15 @@ boolean deployMigrator(Map args = [:]) {
     }
 
     // plan/apply terraform
-    executeTerraform(amiId: amiId,
-                     bfdEnv: bfdEnv,
-                     gitBranchName: gitBranchName,
-                     heartbeatInterval: heartbeatInterval,
-                     createMigratorInstance: true)
+    terraform.deployTerraservice(
+		env: bfdEnv,
+		directory: "ops/terraform/services/migrator",
+		tfVars: [
+            ami_id: amiId,
+            create_migrator_instance: true,
+            migrator_monitor_heartbeat_interval_seconds_override: heartbeatInterval
+		]
+    )
```

Comment:
There's a little tabs-vs-spaces trouble here. Can you make this consistent with the rest of the document?

Reply:
Fixed. Not sure why my vs code environment wasnt reflecting these changes